bot/cogs/movie.py

```diff
@@ -0,0 +1,65 @@
+import requests
+import random
+from discord.ext import commands
+from os import environ
+
+OMDB_API_KEY = environ.get('OMDB_API_KEY')
+
+
+class Movie:
+    """
+    Prints the details of a random scary movie to discord chat
+    """
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name='movie', aliases=['scary_movie'], brief='Pick me a scary movie')
+    async def random_movie(self, ctx):
+        selection = await self.select_movie()
+        movie_details = await self.format_metadata(selection)
+
+        await ctx.send(movie_details)
+
+    @staticmethod
+    async def select_movie():
+        """
+        Selects a random movie and returns a json of movie details from omdb
+        """
+
+        # TODO: Come up w/ a scary movie list to select from. Currently returns random Halloween movie
+        omdb_params = {
+            'apikey': OMDB_API_KEY,
+            'type': 'movie',
+            's': 'halloween'
+        }
+        response = requests.get('http://www.omdbapi.com/', omdb_params)
+
+        movies = []
+        for movie in response.json().get('Search'):
+            movie_id = movie.get('imdbID')
+            movies.append(movie_id)
+
+        selection = random.choice(movies)
+
+        omdb_params = {
+            'apikey': OMDB_API_KEY,
+            'i': selection
+        }
+        response = requests.get('http://www.omdbapi.com/', omdb_params)
+
+        return response.json()
+
+    @staticmethod
+    async def format_metadata(movie):
```

Comment:
I'd like to see this method return an embed instead.

On our PyDis bot, we have a snake movie command, which builds an embed like this with the data it gets from OMDB:

![image](https://user-images.githubusercontent.com/2098517/46601962-20239f00-caef-11e8-8b61-258ab139759e.png)

Would you be able to do something like that?

Reply:
Yeah, that sounds do-able.  Thanks for the feedback.